cypress/integration/landing-page.js

```diff
@@ -0,0 +1,68 @@
+describe('Sol Aware home page', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:3000')
+    })
+
+    it('should have a Header component', () => {
+        cy.get('[data-cy=header]')
+            .should('exist')
+    })
+
+    it('should have a logo with site title', () => {
```

Comment:
I see you are testing that the logo exists on line 13, but do you also want to add a `.should()` for the title so that your tests match your it statement?

Reply:
I see what you are saying, the logo and title are one in the same, so I'll make this change so they are matching - thank you!